cms/djangoapps/contentstore/views/item.py

```diff
@@ -805,8 +812,18 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         "format": xblock.format,
         "course_graders": json.dumps([grader.get('type') for grader in graders]),
         "has_changes": has_changes,
-        "override_type": override_type,
+        "actions": xblock_actions,
+        "explanatory_message": explanatory_message
     }
+
+    # Entrance exam subsection should be hidden. in_entrance_exam is inherited metadata, all children will have it.
```

Comment:
If I am reading your comment correctly, you are saying that if the sequential has "in_entrance_exam" set to True, then its parent will always have "in_entrance_exam" set to True, due to the inheritance. Then why are you also checking the parent status? Can you just delete lines 821-824?